src/v2/view-builder/views/webauthn/ChallengeWebauthnView.js

```diff
@@ -115,8 +148,93 @@ const Body = BaseForm.extend({
   }
 });
 
+const Footer = AuthenticatorFooter.extend({
```

Comment:
Looking at this Footer, I don't think we need to create a complete new one. I think we can just override `links` and return the link we need, otherwise in this new View we are basically copying all the logic that is already in BaseFooter

and a nit: name this `WebAuthnFooter`

Reply:
@mauriciocastillosilva-okta how do you feel about leaving the new initialize() function?
there are 2 things different here:
one is that we use `getBackToSignInLink()` instead of `getSignOutLink()`
and we have the extra logic to add the new view when we add the link

Reply:
Why do this view has a "back to sign in" link instead of a "sign out" like the other challenge views? This will mean that if I am in an authentication flow using say Phone and OV, then Phone will show "sign out" and OV will show "back to sign in"? That sounds inconsistent, can you please check?

I am still in favor of not duplicating the logic of BaseFooter, and I think we can get away with enhancing Link to support a "toggle text link". Option B can be to refactor BaseFooter and have functions for the common code so we avoid duplicating it.

Reply:
see `IdentifierFooter.js ` for example.

Reply:
@pradeepdewda-okta i don't think `footerInfo` works for what i'm needing to do here..
it looks like `footerInfo` adds an extra section at the VERY end of the footer..
i'm just trying to implement a sort of expand/collapse - take a look at the screenshots in the PR description

Reply:
@mauriciocastillosilva-okta it looks like other authenticators also have `Back to sign in`

<img width="405" alt="Screen Shot 2021-06-28 at 1 28 37 PM" src="https://user-images.githubusercontent.com/19613909/123700032-ce7a5000-d814-11eb-9786-2ff26248c80c.png">

<img width="405" alt="Screen Shot 2021-06-28 at 1 29 49 PM" src="https://user-images.githubusercontent.com/19613909/123700178-f9fd3a80-d814-11eb-89a1-8515eaf342bf.png">

Reply:
@pradeepdewda-okta this is what i'm trying to do - i think this is very different behavior than what footerInfo is intended to do

![2021-06-28 13 48 10](https://user-images.githubusercontent.com/19613909/123702211-b3f5a600-d817-11eb-9a1e-69e36887d183.gif)

Reply:
@jasleennuno-okta you right for this you have to enhance `Links.js` https://github.com/okta/okta-signin-widget/blob/master/src/v2/view-builder/components/Link.js

you can add another param similar to `footerInfo` say.. `linkInfoView` that could be passed as a view from your `Footer` class.. `Link.js` can add such a view under any link..

Reply:
discussed offline with @jasleennuno-okta . This is the plan
- Enhance `Links.js` so it supports a way to add a view inside the link that can be toggled. This is similar to what Jasleen had initially, but we will move the logic to `Links.js` instead of been in the Footer. This will also make it generic for any other view to consume.
- `linkInfoView` or something similar sounds good to me, and we will need a small css override for that new view to not inherit the link styles
- Remove the override of Footer once we confirm that `getSignOutLink` works properly, which it should since we have a cancel remediation form, but need to check details

@pradeepdewda-okta we can't use `footerInfo` for the reasons Jasleen mentioned, but we should be good with the plan I just described

Reply:
@mauriciocastillosilva-okta sounds about right. thanks @jasleennuno-okta for improving widget code..